packages/streams/package.json

```diff
@@ -1,11 +1,11 @@
+
```

Comment:
`insert_final_newline` is currently set to `true` in the `.editorconfig` - maybe we should change that?'
[Relevant line](https://github.com/material-motion/material-motion-js/blob/d07b8a91ed2be5a2ccfb8e3f00390c05e0744c2f/.editorconfig#L10).